azure/04-cluster-services/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as azure from "@pulumi/azure";
+import * as pulumi from "@pulumi/pulumi";
+import { config } from "./config";
+
+const name = pulumi.getProject();
+
+// Enable the Monitoring Diagonostic control plane component logs and AllMetrics
+const azMonitoringDiagnostic = new azure.monitoring.DiagnosticSetting(name, {
+    logAnalyticsWorkspaceId: config.logAnalyticsWorkspaceId,
+    targetResourceId: config.clusterId,
+    logs: [{
+            category: "kube-apiserver",
```

Comment:
Let's enable all logs for the control plane per step #6: https://docs.microsoft.com/en-us/azure/aks/view-master-logs#enable-diagnostics-logs

Reply:
IIUC this also sets up logging for containers & events, right? Can we double check this is showing up in the appropriate Azure console pages per https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-live-logs

We'll want to mention this in the docs portion of this stack.